app/resources/views/pages/location_report/collapsed.blade.php

```diff
@@ -22,8 +22,9 @@
 		'location' => $location))
 	<div class="basic-info">
 		<div class="addresses">
-			<address>{{ $location->address }}</address>
-			<a href="{{ $location->getExternalWebURL() }}">{{ $location->getExternalWebURL() }}</a>
+		<p><em class="fa fa-map-marker"></em>{{ $location->address }}</p>
```

Comment:
Keep using the "address" tag name for the address because it is good for search engine optimization.  The "address" tag is meaningfully matched with the text it contains.  If you were trying to style the address to have a different margin-right, you can do so in the _location_map_report.scss.